jsFiles/utility.js

```diff
@@ -46,19 +46,26 @@ function dateTimeDisplay(date) {
 
 /**
  * This function converts timestamp or a date object,
- * adds the offset time and returns a displayable time --> hh:mm AM/PM
+ * adds the offset time and returns a time in minutes or
+ * a displayable time --> hh:mm AM/PM
  * Currently use for the sunrise and sunset time stamps
  * and to display the time of the searched city
  *
  * @param {number or object} dateInfo - unix, contains information about the time.
  * @param {number} offsetTime - is the seconds a certain time zone is ahead of or behind UTC.
+ * @param {boolean} minutesFormat - false default, true when the time is needed in minutes
  */
-function timeConversion(dateInfo, offsetTime) {
-  const time =
+
+function timeConversion(dateInfo, offsetTime, minutesFormat = false) {
```

Comment:
I'll call it `inMunites` or `shouldReturnInMinutes`. For me it's easier to call flags with something which speak for itself.